esp/src/eclwatch/ESPWorkunit.js

```diff
@@ -312,7 +312,15 @@ define([
             WsWorkunits.WUUpdate({
                 request: request,
                 load: function (response) {
-                    context.updateData(response.WUUpdateResponse.Workunit);
+                    if (lang.exists("Exceptions.Exception", response)) {
```

Comment:
The issue with the toaster in ESPRequest is that the message for this particular issue is coming from the error stack is not useful but the exceptions message is.